### ISSUE TITLE
Enable secureboot images on AWS (stage0 secureboot)

### ISSUE THREAD

**Issue by pnpavlov** [gardenlinux/gardenlinux#2232]
As Garden Linux user, I want to have an image with secureboot enabled out of the box on AWS. I should have a clear log informing me that secureboot has been enabled.

**Comment by nkraetzschmar** [comment]
https://github.com/gardenlinux/gardenlinux/pull/2237

**Comment by pnpavlov** [comment]
Feature ready. We postpone the release until Sept, @MrBatschner  to follow up with Gardener team on the adoption.

**Comment by MrBatschner** [comment]
Currently blocked by a bug in AWS' TPM architecture, need to wait until they fix their stuff.

**Comment by pnpavlov** [comment]
@MrBatschner , can we take that forward next week?

**Comment by pnpavlov** [comment]
@nkraetzschmar @MrBatschner , to my understanding this issue is completed and can be closed, can you please confirm and close if implementation is done and tests are in place?

**Comment by pnpavlov** [comment]
done.platform tests running with nightly.
